platform/docs/docs/deployment/build-for-production.md

```diff
@@ -108,6 +108,10 @@ yarn global add http-server
 npx http-server ./dist
 ```
 
+:::caution
+In the video below you see there is `platform/viewer` which has been renamed to `platform/app` in the latest version
```

Comment:
Instead of using "you", I prefer something like...

"In the video below notice that there is..."

But that is just my preference/opinion. I feel like it is more professinal, but "you" is fine. Just wanted to point it out.